### PR TITLE
Schinagl.LinkShellextension version 3.9.3.5

### PR DESCRIPTION
**Pull request by Masamune3210**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/55883)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
The installer requires VC++2015-2019 to be installed before running, needs to wait for dependency support
InstallationVerification-x64-machine-Locale_Undefined-ErrorScreenshot_20220326205848
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20220326205620

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/55883#issuecomment-1084206515

File: manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Schinagl.LinkShellextension
+PackageVersion: 3.9.3.5
+PackageLocale: en-US
+Publisher: Hermann Schinagl
+PackageName: Link Shell Extension
+License: Copyright © 1999-2020
```
Comment (Trenly) [inline manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml:9]: A copyright is not a license

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Masamune3210** [comment]
Waiting for external dependency support

**Review by ItzLevvie — APPROVED** [review]
Just approving for filtering purposes.

At the moment package dependency support still hasn't gone out of experimental features. So you probably won't see this until somewhere in Q3 or Q4 of 2022 or somewhere in 2023.

**Comment by Masamune3210** [comment]
Just undrafting things to run them through the validator, is dependency support out yet?

**Comment by OfficialEsco** [comment]
IIRC no, they hit some (a lot) speedbumps so they didn’t release any Dependency features yet, even in 1.3 its hidden behind the feature command. (Also pretty much no changes refering to Dependcies)

**Comment by Masamune3210** [comment]
Ok, back to draft it goes

**Pull request closed without merging**